scalableminds/webknossos
# Fix max of empty array error when resizing wk during loading

…and some other minor improvements/clean ups

### URL of deployed dev instance (used for testing):
- https://maxofemptyarrayfix.webknossos.xyz

### Steps to test:
- open an annotation or dataset
- when the loading screen is shown, resize the window
- --> no error should occur

### Issues:
- fixes occasional "max of empty array" reports in airbrake

------
(Please delete unneeded items, merge only when none are left open)
